daemon/create.go

```diff
@@ -150,6 +150,14 @@ func (daemon *Daemon) generateSecurityOpt(ipcMode containertypes.IpcMode, pidMod
 
 		return label.DupSecOpt(c.ProcessLabel), nil
 	}
+	if pidContainer := pidMode.Container(); pidContainer != "" {
+		c, err := daemon.GetContainer(pidContainer)
+		if err != nil {
+			return nil, err
+		}
+
+		return label.DupSecOpt(c.ProcessLabel), nil
```

Comment:
You could release note it.  Not sure if their would be other problems.  Combining containers together without combining their security attributes could lead to some weird errors.  Right now we keep seccomp, apparmor and capabiltiies separate, but you could figure out ways where weird bugs or priv escalations can happen.  Basically one process with limited privs in one container causing a process in a different container through IPC or shared pid to do something more privileged.